tests/create_policies/main.tf

```diff
@@ -1,39 +1,53 @@
-provider aws {
+provider "aws" {
   region = "us-east-1"
 }
 
-module policies {
-  source   = "../..//modules/policy"
+module "policies" {
+  source   = "../../modules/policy"
```

Comment:
You switch to a single `/` for the relative path in some places but not others.  Any reason why?